stable/kibana/values.yaml

```diff
@@ -17,11 +17,11 @@ env: {}
   # SERVER_DEFAULTROUTE: "/app/kibana"
 
 files:
-  kibana.yml:
+  kibana.yml: {}
```

Comment:
I've just checked my kibana.yaml.

It looks like this:

```
elasticsearch.url: http://elasticsearch:9200
server.host: "0"
server.name: kibana
```

Looks totally ok to me, as its exactly that what was configured.

So where exactly comes this merge error from you're referring to in https://github.com/helm/charts/issues/12199

Reply:
setting `elasticsearch.hosts` will produce kibana config error and setting `elasticsearch.url` as `null` or empty string `""` doesn't help.

As default config in this chart is exactly same with [kibana-docker's default config](https://github.com/elastic/kibana-docker/blob/master/.tedi/template/kibana.yml.j2), this default config won't do anything but making this deprecated config error

Reply:
#12378